instructor/common.py

```diff
@@ -274,7 +282,9 @@ def assignments_variables(request):
     form = AssignmentForm(instance=assignment)
     return render_to_response('instructor/assignment_select_variables.html',
                               {'form': form,
-                               'assignment': assignment
+                               'treatments_created': treatments_created,
+                               'assignment': assignment,
+                               'assignment_name': assignment.name
```

Comment:
Nevermind, wasn't thinking. But you should still probably remove `assignment_name` since you can use `assignment.name` in the template